pkg/landscaper/controllers/deployitem/reconcile_new.go

```diff
@@ -193,8 +195,9 @@ func (con *controller) writeAbortingTimeoutExceeded(ctx context.Context, log log
 	return nil
 }
 
-func (con *controller) isProgressingTimeoutExceeded(log logging.Logger, di *lsv1alpha1.DeployItem) (bool, *time.Duration, error) {
-	logger := log.WithValues("operation", "DetectProgressingTimeouts")
+func (con *controller) isProgressingTimeoutExceeded(ctx context.Context, di *lsv1alpha1.DeployItem) (bool, *time.Duration, error) {
+	logger, _ := logging.FromContextOrNew(ctx, nil)
+	logger = logger.WithValues(lc.KeyMethod, "isProgressingTimeoutExceeded")
```

Comment:
```suggestion
	logger, ctx := logging.FromContextOrNew(ctx, nil, lc.KeyMethod, "isProgressingTimeoutExceeded")
```

Reply:
Again the `KeyMethod` logging key, which is only for the current method, but should not be passed in a context to sub methods.